secp256k1-sys/vendor-libsecp.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+
+
+if [ -z "$1" ] | [ -z "$2" ]; then
+  echo "\$1 parameter must be the rust-secp256k1-sys depend directory"
+  echo "\$2 parameter must be the rust-secp256k1-sys version code (M_m_p format)"
+  echo "\$3 parameter (optional) can be the revision to check out"
+  exit 1
+fi
+
+PARENT_DIR=$1
+VERSIONCODE=$2
+REV=$3
+DIR=secp256k1
+ORIGDIR=$(pwd)
+
+while true; do
+    read -r -p "$PARENT_DIR/$DIR will be deleted [yn]: " yn
+    case $yn in
+        [Yy]* ) break;;
+        [Nn]* ) exit;;
+        * ) echo "Please answer yes or no.";;
+    esac
+done
+
+cd "$PARENT_DIR" || exit 1
+rm -rf "$DIR"
+git clone git@github.com:bitcoin-core/secp256k1.git "$DIR"
```

Comment:
Can't you just clone over https? This does not work for me because it wants a ssh key.

Reply:
oh, yeah that makes sense.

Reply:
I pushed an extra commit.